<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>251</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="frames">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>8</number>
    </property>
    <widget class="QWidget" name="circle">
     <widget class="QLabel" name="customCircle">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your circle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="changeRadius">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Change radius:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="radiusSpinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rectangle">
     <widget class="QLabel" name="customRectangle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your rectangle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="heightRectangle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QLabel" name="widthRectangle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="widthrectspinbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>2.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="heightrectspinbox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Right_polygone">
     <widget class="QDoubleSpinBox" name="spinboxside_rd">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="numberOfSides">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of sides:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinboxnumapexes_rp">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="changeSides">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the sides:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your polygone:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rhombus">
     <widget class="QDoubleSpinBox" name="spinbox_angle_rh">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinbox_side_rh">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="changeSides_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the sides:</string>
      </property>
     </widget>
     <widget class="QLabel" name="changeAngle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the angle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="customRhombus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your rhombus:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="square">
     <widget class="QLabel" name="customSquare">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your square:</string>
      </property>
     </widget>
     <widget class="QLabel" name="changeSides_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the sides:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="star">
     <widget class="QDoubleSpinBox" name="radius1_spinbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="radius2_spinbox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="customStar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your star:</string>
      </property>
     </widget>
     <widget class="QLabel" name="widthApexes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width of apexes:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lengthApexes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Length of apexes:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="apexes_spinbox">
      <property name="geometry">
       <rect>
        <x>153</x>
        <y>40</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="numberOfApexes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of apexes:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="triangle">
     <widget class="QLabel" name="customTriangle">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Custom your triangle:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sideOne">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Side One:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sideTwo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Side Two:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sideThree">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Side Three:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="side1trianglespin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="side2trianglespin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="side3trianglespin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>65</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Clearscene"/>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="drawButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>88</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>DRAW</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose figure...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right Polygone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rhombus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Star</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clear scene</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="rotate_frame">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="rotationcenterXspinbox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>65</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotationcenterYspinbox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>65</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="rotationX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rotationY">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rotationPoint">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotation about a point:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Changeframe">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="adjustments">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>-1</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Adjustments:</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_label3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>69</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Perimeter:</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_label2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Area:</string>
     </property>
    </widget>
    <widget class="QLabel" name="calculateArea_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="calculatePerimeter_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sizespinbox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotationspinbox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>800.990000000000009</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="cahgexspinbox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-200.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="changeyspinbox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-150.000000000000000</double>
     </property>
     <property name="maximum">
      <double>150.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="moveX">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Move (x):</string>
     </property>
    </widget>
    <widget class="QLabel" name="moveY">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Move (y):</string>
     </property>
    </widget>
    <widget class="QLabel" name="rotateAngle">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate:</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="connectCenterButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Click here to connect the point with center</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
